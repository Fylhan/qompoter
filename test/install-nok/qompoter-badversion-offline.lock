{
  "name": "test/install-nok-badversion",
  "date": "2016-12-02T18:24:51+01:00",
  "require": {
  }
}
