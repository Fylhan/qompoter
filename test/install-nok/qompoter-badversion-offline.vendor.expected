vendor
vendor/qompote.pri
vendor/vendor.pri
